.github/workflows/build.yml

```diff
@@ -138,6 +138,13 @@ jobs:
       with:
         toolchain: stable
 
+    # temporarily necessary while skia_bindings 0.78.2 is looking for the `pipes` module which
```

Comment:
Yep... I had to do this also. 🙄  On Mac GitHub now runs 3.13 by default, even though their specs say 3.12.  The other OSs were still on 3.12, but better safe than sorry.

Reply:
PS. @samizdatco  Sorry, I didn't realize this error came up due to adding `webp` in the first place. I had thought it was just a new requirement for the updated skia-safe in general. Though in retrospect your test workflows didn't need that before this change, which should have been a clue for me. Thanks for fixing.